src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -44,12 +44,11 @@ public static void loadJni() {
     return;
   }
 
-  /** Returns a new {@link ChipDeviceController} with default parameters. */
-  public ChipDeviceController() {
-    this(ControllerParams.newBuilder().build());
-  }
-
   /** Returns a new {@link ChipDeviceController} with the specified parameters. */
+  /**
+   * you must set a vendor ID, ControllerParams.newBuilder().setControllerVendorId(0xFFF4).build()
+   */
+  /** 0xFFF4 is a test vendor ID */
```

Comment:
Doc comments should be grouped as one unit rather than split up.

Consider:

```java
/**
  Returns a new {@link ChipDeviceController} with the specified parameters.
   
  Parameters are expected to be fully configured, in particular vendorID MUST be set.
 */